Character.h

```diff
@@ -0,0 +1,28 @@
+#include <string>
+#include <iostream>
+#ifndef CHARACTER_CLASS_H
+#define CHARACTER_CLASS_H
+
+
+class Character
+{
+private:
+	int hp; const int  damage;
+	const std::string name;
+
+public:
+	//constructor
```

Comment:
ezek a kommentek igazabol csak bloatolnak, nem segitenek. Nem baj, csak megjegyzes. egy getter ele pl ne kelljen kiirni, hogy ez egy getter, az a sajat nevebol hadd jojjon le. :-)

Reply:
Szintén a többieknek szólt, de rájönnek nélküle is :D

Reply:
Vilagos, de az a jo, ha a "kod magaert beszel". Nem egy profi (en nem vagyok az), hogy a legjobb az, ha nem kommentelunk. Ha kommentelni kell, akkor rosszul van megirva a kod.